src/main/java/com/github/manolo8/darkbot/backpage/NovaManager.java

```diff
@@ -0,0 +1,205 @@
+package com.github.manolo8.darkbot.backpage;
+
+import com.github.manolo8.darkbot.backpage.nova.Agent;
+import com.github.manolo8.darkbot.backpage.nova.NovaData;
+import com.github.manolo8.darkbot.backpage.nova.Perk;
+import com.github.manolo8.darkbot.utils.http.Method;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+
+import java.io.IOException;
+
+public class NovaManager {
+    private final BackpageManager backpageManager;
+    private final NovaData data;
+    private final Gson g;
+    private long lastNovaUpdate;
+
+    public NovaManager(BackpageManager backpageManager) {
+        this.backpageManager = backpageManager;
+        this.data = new NovaData();
+        this.g = new Gson();
```

Comment:
any reason for using a new gson instance here?
we have a few scattered around the codebase by now, probably should try to reuse them more